DigitalLearningSolutions.Web/Models/DelegateClaimDetails.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DigitalLearningSolutions.Web.Models
+{
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public class DelegateClaimDetails
```

Comment:
Could we name these classes to DelegateLoginDetails and AdminLoginDetails as I think that's a bit more intuitive?

Reply:
I'd hesitate on the "...Login..." part because to me that would imply it contains details for logging in (e.g. password), but then I can't think of a better name, so I'll go with your suggestions.